plugins/services/src/js/containers/tasks/TaskTable.js

```diff
@@ -394,6 +394,7 @@ class TaskTable extends React.Component {
         sortBy={{ prop: "updated", order: "desc" }}
         sortOrder="desc"
         sortProp="updated"
+        uniqueProperty="id"
```

Comment:
That was removed for a reason here https://github.com/dcos/dcos-ui/commit/69a2560957c884aa83319713f14cf304ebd8c589 These IDs are not unique which breaks other features. I wonder if we should summon the Mesos/Marathon gods to fix the source of the problem @MatApple @orlandohohmeier